doc/source/reference/c-api.ufunc.rst

```diff
@@ -23,6 +23,12 @@ Constants
 
     ``{VALUE}`` can be **One** (1), **Zero** (0), or **None** (-1)
 
+    .. c:var:: PyUFunc_One
+
+    .. c:var:: PyUFunc_Zero
+
+    .. c:var:: PyUFunc_None
+
```

Comment:
To make it a bit trickier: Can you check this function:
https://github.com/numpy/numpy/blob/d94acc7f73795ecbec71b672fd7c5f07e0ce9f09/numpy/core/src/umath/ufunc_object.c#L2463

and try to capture that in the documentation here? This code was changed fairly recently and the updates never made it into the docs.

Reply:
I suppose if we list them all, we can maybe delete that line about `{VALUE}` being one of, but just a thought.